// config/mainconfig.js
export const MainConfig = {
  AppDetails: {
      app_name: "E-Beta Drive", 
      app_description: "E-Beta Drive For Taxi Booking",
      app_identifier: "com.webgarh.ebetadrive",
      ios_app_version: "1.0.0", 
      android_app_version: 100
  },
  FirebaseConfig: {
      apiKey : "AIzaSyAMproU_DWvK7WJqiOT_sF22CA-yXsAJUM" , 
  authDomain : "ebeta-drive.firebaseapp.com" , 
  databaseURL : "https://ebeta-drive-default-rtdb.firebaseio.com" , 
  projectId : "ebeta-drive" , 
  storageBucket : "ebeta-drive.appspot.com" , 
  messagingSenderId : "1024346069699" , 
  appId : "1: 1024346069699: web: 16731edf49150ca354e076" , 
  measurementId : "G-2C2NVWEMQZ"
  },
  Google_Map_API_Key: {
      ios: "AIzaSyCdgwTpKmUnlFAAi7G4BwhspB9bLZtQpSM",
      android: "AIzaSyCdgwTpKmUnlFAAi7G4BwhspB9bLZtQpSM",
      web: "AIzaSyCdgwTpKmUnlFAAi7G4BwhspB9bLZtQpSM"
  },
  facebookAppId: "234877565373142",
  PurchaseDetails: {
      CodeCanyon_Purchase_Code: "b20a06dd-06cf-4544-8a71-c918bd473c6f",
      Buyer_Email_Address: "ebetadrive@gmail.com"
  }
}
